blog/2023-12-22-openhab-4-1-release.md

```diff
@@ -59,6 +59,33 @@ This can be used for example to add a location or equipment that does not (yet)
 To define your own semantic tags, create a YAML configuration file containing the description of all your specific semantic tags in the `$OPENHAB_CONF/tags` folder.
 You can find a [tutorial on the community forum](https://community.openhab.org/t/oh-4-1-tutorial-to-manage-custom-semantic-tags/148135).
 
+### Voice System Enhancements
+
+_Miguel Álvarez Díez, openHAB Contributor_
+
+Some minor improvements were introduced in the built-in voice system components. 
+
+The standard text interpreter now understand your Item synonyms, 
+and gives priority to an exact match on the label/synonym allowing an easier control over the command target.
+It also understand your Item metadata descriptions, 
+the rule `Set <ItemLabel> to <ItemCommandLabel>` is available out-of-the-box for the supported languages for all the Items with the commandDescription metadata.
```

Comment:
What do you think about? And is this correct?

> It also automatically scans your Items` command description metadata, so that the rule `Set <ItemLabel> to <ItemCommandLabel>` is supported out-of-the-box for all supported languages.

Out of interest: Without having command description metadata on a Switch Item, is `turn on <ItemLabel>` supported?

Reply:
Yes.

It just affects to that rule, and the rule it's only understood when targeting the items with that metadata, meaning if you have two slider items labeled as "A" and "B" and only "B" has the item command description "100=high" only "set b to high" will work, "set a to high" will told you an error response. If you add then add an item labeled as "C" with commands description "90=high" it will also be understood and send the command 90 instead of 100.